lib/modulesync.rb

```diff
@@ -39,10 +39,10 @@ def self.find_template_files(local_template_dir)
           .collect { |p| p.chomp('.erb') }
           .to_a
     else
-      $stdout.puts "#{local_template_dir} does not exist." \
+      warn "#{local_template_dir} does not exist." \
```

Comment:
This is [Kernel.warn](https://apidock.com/ruby/Kernel/warn), right? I'm not sure if that's the correct thing to do. Other places in this file use `$stdout.puts` and `$stderr.puts` so I'd prefer to remain consistent. In this case I think it should be `$stderr.puts`. The same below.

The reason is that you can disable all warnings when you run `ruby -W0` which I sometimes do if there's some deprecation warning. Some projects generate them because they're fully Ruby 2.7 clean. That would hide this output.

Reply:
My wish was to write clean code and `rubocop` recent version advises to turn `$stderr.puts` int `warn` but you're right: `warn` can be disabled using `ruby -W0`.
So, I will revert the turn from `$stderr.puts` into `warn` as its more an error than a warning.
Thanks for notice.

Source: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/StderrPuts